lib/models/application.coffee

```diff
@@ -85,14 +85,19 @@ getApplicationModel = (deps, opts) ->
 	exports.getAll = (options = {}, callback) ->
 		callback = findCallback(arguments)
 
-		token.getUserId().then (userId) ->
+		request.send
+			method: 'GET'
+			url: '/user/v1/whoami'
+			baseUrl: apiUrl
+		.get('body')
+		.then (body) ->
```

Comment:
This should reuse the existing code in auth to get the user id, rather than duplicating it here.

As above - we should also avoid making an extra request here, since this is very frequently used code, so we'll need to look at caching for this.